### ISSUE TITLE
Repetitive Logic in Bank Actions (fd, rd, sip, etc.)

### ISSUE THREAD

**Issue by VinVorteX** [isatyamks/CodeCash#16]
Issue: The logic for handling bank actions (fd, rd, sip, lumpsum, loan) is repetitive. This not only increases the chances of bugs but also makes the code harder to maintain.

Solution: Refactor the logic using a dictionary that maps the action_type to a function, reducing repetition.

**Comment by isatyamks** [comment]
i cant assign it to you feel free to work on this issue.

**Comment by VinVorteX** [comment]
Yes but as I can see, the issues you opened before are all assigned and I am doing it for hacktober fest, I will work on this issue but I will not get any score regarding this in hacktober
